ams/optimizer/src/main/java/com/netease/arctic/optimizer/LocalOptimizerContainer.java

```diff
@@ -54,7 +53,7 @@ protected String buildOptimizerStartupArgsString(Resource resource) {
     long memoryPerThread = Long.parseLong(PropertyUtil.checkAndGetProperty(resource.getProperties(),
         JOB_MEMORY_PROPERTY));
     long memory = memoryPerThread * resource.getThreadCount();
-    return String.format("%s/bin/localOptimize.sh -m %s %s", getAMSHome(), memory,
+    return String.format("%s/bin/localOptimize.sh %s %s", getAMSHome(), memory,
```

Comment:
After remove `-m`, The unit test cases in `TestOptimizerConfig` also needs adjustment.

Reply:
Done.